.github/workflows/cs-fix.yml

```diff
@@ -9,7 +9,8 @@ on:
 jobs:
   php-cs-fixer:
     runs-on: ubuntu-latest
-    if: github.event.pull_request.draft == false
+    # dont run jobs on forks, because we are not allowed to commit
+    if: github.event.pull_request.draft == false && github.repository == 'redaxo/redaxo'
```

Comment:
`github.repository` ist das Repository, indem die Action läuft. Bei uns also immer redaxo/redaxo.
(In dem verlinkten Community-Thread ging es wohl darum, die Actions innerhalb des Forks nicht ablaufen zu lassen. Es ging also nicht um PRs von Forks zum Hauptrepo.)

Reply:
D.h.?

Reply:
Dass dieser PR keine Auswirkung hatte, und ich aber auch nicht weiß, wie man das eigentlich gewünschte Ziel stattdessen erreichen kann.